### PR TITLE
Rearrange the running tasks.

### PR DESCRIPTION
**Pull request by milose**
I made a few tweaks:

- Git commit should be run after we are done with `.env` and `.env.example`.
- Only the `.env` file should be manipulated by `perl`, and later copied to `.env.example` file.
- After script should run before browser is opened.

### CODE REVIEW

File: lambo [hunk 1]
```diff
@@ -363,6 +367,15 @@ if [[ "$SECURE" = true ]]; then
     valet secure "$PROJECTNAME"
 fi
 
+### Load after file if it exists
```
Comment (mattstauffer) [inline lambo]: This is a good change 👌

File: lambo [hunk 2]
```diff
@@ -351,10 +350,15 @@ perlCommands=(
 )
 
 for perlCommand in "${perlCommands[@]}"; do
-    perl -pi -e "$perlCommand" ".env.example"
     perl -pi -e "$perlCommand" ".env"
 done
 
+cp .env .env.example
```
Comment (mattstauffer) [inline lambo]: I know it doesn't *really* matter, but I think I'd rather we do the modifications to the `.env.example` file and then copy it to `.env`. Technically it'll be exactly the same but it makes a little bit more sense to my brain :)

**Comment by milose** [comment]
Umm. This is not good yet. App key is not generated. Give me a sec.

**Comment by milose** [comment]
Should we remove this ambiguous copy operation and run `php artisan key:generate --ansi` before commit,  instead? @mattstauffer

**Comment by mattstauffer** [comment]
Yah, I would rather we avoid putting the app key in version control.

**Comment by milose** [comment]
I made the change. It all works now. The `key:generate` is run once before, by `laravel new` but I'd rather have it like this, with easy to read steps, than further complicating the script by extracting the key that was generated earlier.

**Comment by mattstauffer** [comment]
Thanks @milose!

**Pull request merged**